[submodule] CUTLASS upgrade to 4.2.0 and change cutlass to cutlass_cppgen

henrylhtsang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #163092

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov @coconutruben

henrylhtsang committed:
change cutlass to cutlass_cppgen



Differential Revision: [D81520542](https://our.internmc.facebook.com/intern/diff/D81520542/)

[ghstack-poisoned]

henrylhtsang commented:
https://github.com/pytorch/pytorch/pull/162348

henrylhtsang commented:
fyi @Skylion007 there is a change I need to make with regards to the python cutlass name

drisspg reviewed (approved):
Are the tests working w/ pip install cutlass now?

henrylhtsang commented:
> Are the tests working w/ pip install cutlass now?

afaik, no. But that is because the pip install PR hasn't merged yet

henrylhtsang committed:
Update on "[submodule] CUTLASS upgrade to 4.2.0 and change cutlass to cutlass_cppgen"


Differential Revision: [D81520542](https://our.internmc.facebook.com/intern/diff/D81520542/)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov coconutruben

[ghstack-poisoned]

henrylhtsang committed:
Update on "[submodule] CUTLASS upgrade to 4.2.0 and change cutlass to cutlass_cppgen"


Differential Revision: [D81520542](https://our.internmc.facebook.com/intern/diff/D81520542/)

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov coconutruben

[ghstack-poisoned]

Skylion007 reviewed (approved):
Awesome

henrylhtsang committed:
Update on "[submodule] CUTLASS upgrade to 4.2.0 and change cutlass to cutlass_cppgen"


cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov coconutruben

[ghstack-poisoned]

henrylhtsang committed:
Update on "[submodule] CUTLASS upgrade to 4.2.0 and change cutlass to cutlass_cppgen"


cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov coconutruben

[ghstack-poisoned]

henrylhtsang commented:
@pytorchbot merge
